terra_notebook_utils/blobstore/copy_client.py

```diff
@@ -0,0 +1,100 @@
+import os
+import logging
+import multiprocessing
+from concurrent.futures import ProcessPoolExecutor
+from typing import Union
+
+from getm import default_chunk_size
+from getm.concurrent.collections import ConcurrentHeap
+
+from terra_notebook_utils.blobstore.local import LocalBlobStore, LocalBlob
+from terra_notebook_utils.blobstore.gs import GSBlobStore, GSBlob
+from terra_notebook_utils.blobstore import BlobstoreChecksumError
+
+
+logger = logging.getLogger(__name__)
+
+AnyBlobStore = Union[LocalBlobStore, GSBlobStore]
```

Comment:
Guessing the URLBlobstore addition will be a separate PR?

Reply:
ugh will add now